hack/generated/apis/microsoft.documentdb/versions_matrix.md

```diff
@@ -0,0 +1,56 @@
+|                                                               | v1alpha1api20210515 |
+|---------------------------------------------------------------|---------------------|
```

Comment:
Do we actually want the version matrix files committed? I sorta thought they were just for our consumption? Maybe we should gitignore `md` files in `apis`?

Reply:
No, I always intended that these would be committed as a reference for which types have been generated in each specific version. I've got a ToDo to work out why the headers are now being omitted from the tables.